spec/features/ephemera_folder_spec.rb

```diff
@@ -180,17 +176,6 @@
           expect(page.find(:css, ".alert-info")).to have_content "Deleted EphemeraFolder"
         end
 
-        scenario "users see a warning if they try to use duplicate barcodes" do
```

Comment:
Note for reviewers: This got removed because it's a duplicate test in this file.